test/controllers/website/CaptureWebsiteControllerSpec.scala

```diff
@@ -95,47 +95,15 @@ class CaptureWebsiteControllerSpec extends ControllerBaseSpec {
 
     "there is no website in session" when {
 
-      "the customerInfo call succeeds" should {
-
-        lazy val result = {
-          mockVatSubscriptionCall()
-          target().show(request)
-        }
-        lazy val document = Jsoup.parse(bodyOf(result))
-
-        "return 200" in {
-          status(result) shouldBe Status.OK
-        }
-
-        "return HTML" in {
-          contentType(result) shouldBe Some("text/html")
-          charset(result) shouldBe Some("utf-8")
-        }
+      lazy val result = target().show(request)
```

Comment:
There's a function `mockVatSubscriptionCall` in this spec that is now unused and can be removed